tasks/go.py

```diff
@@ -391,6 +411,40 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg in additional:
+        if pkg.startswith(GITHUB_DOMAIN):
+            url = "https://{}/master/".format(pkg.replace(GITHUB_DOMAIN, GITHUB_RAW_DOMAIN))
+        else:
+            url = "https://{}".format(pkg)
+
+        for filename in LICENSE_FILES:
+
+            url = urljoin(url, filename)
+            try:
+                resp = requests.get(url)
+                resp.raise_for_status()
+
+                with tempfile.TemporaryDirectory() as tempdir:
+                    with open(os.path.join(tempdir, filename), 'w') as lfp:
+                        lfp.write(resp.text)
+                        lfp.flush()
+
+                        temp_path = os.path.dirname(lfp.name)
+                        result = ctx.run("go run github.com/go-enry/go-license-detector/v4/cmd/license-detector -f json {}".format(temp_path), hide='err')
+                        if result.stdout:
+                            results = json.loads(result.stdout)
+                            for project in results:
+                                if 'error' in project:
+                                    continue
+
+                                # we get the first match
+                                license = project['matches'][0]['license']
+                                licenses.append("core,\"{}\",{}".format(pkg, license))
+            except Exception as e:
```

Comment:
I am not a fan of catching all exceptions; if the task breaks in some unexpected way it will be difficult to know it did. Can we catch a more restricted type of exceptions?